hms-app/src/components/HackathonSelectDropdown.tsx

```diff
@@ -27,38 +31,63 @@ export default function HackathonSelectDropdown(setHackathonID: Props) {
     )
   }
 
-  const hackathonMap = hackathonList.map((hackathon, index) => ({
-    value: hackathon.id,
-    label:
-      hackathon.title +
-      ' ' +
-      new Date(hackathon.startDate).toLocaleDateString() +
-      '-' +
-      new Date(hackathon.endDate).toLocaleDateString(),
-  }))
+  const futureAndPresentHackathons = hackathonList.filter((hackathon) => {
+    return new Date(hackathon.endDate) > today
+  })
 
-  const selectChange = (value: string) => {
-    setSelectedHackweek(value)
+  const pastHackathons = hackathonList.filter((hackathon) => {
```

Comment:
so you shouldn't create both of these variables.  At best you could lazy load into variables, but likely this part you are doing should be done within the `getHackathonsForContext()` funciton

Reply:
Oh.. nice catch 🚀  When I went trough it earlier, i tough these were functions accepting `hackatonList` as parm. In this case you need to either make them functions or use them directly as expressions in `getHackathonsForContext()` directly.